### PR TITLE
TR-1781 - Fix creating new template in V2

### PR DESCRIPTION
**Pull request by nicklemmon**
[TR-1781](https://jira.int.messagesystems.com/browse/TR-1781)

### What Changed
- Text content passed to create method will travel with `POST` as the API does not support a null value for the text field
- This was introduced as the result of a prior bug fix for the original templates implementation

### How To Test
- Go to `/templatesV2` and click 'Create New'
- Complete the form and click 'Next'

### To Do
- [ ] Incorporate feedback
- [X] Add supporting test to check that request is passing along an empty string via `content.text` < -- Nevermind - updating the existing test accomplishes this as the `CreatePage.test.js` is already making this check

### CODE REVIEW

File: src/actions/helpers/templates.js [hunk 1]
```diff
@@ -18,8 +18,8 @@ export const getTestDataKey = ({ id, username, mode }) => ([ 'tpldata', username
 export const shapeContent = (content = {}) => {
   const shapedContent = {
     ...content,
+    text: content.text,
```
Comment (bkemper) [inline src/actions/helpers/templates.js]: Maybe add a comment for why we allow an empty text part?

Comment (nicklemmon) [reply src/actions/helpers/templates.js]: That definitely makes sense
